modules/nw-aws-local-zones-user-workload-ingress.adoc

```diff
@@ -0,0 +1,9 @@
+:content-type: CONCEPT
+[id="nw-aws-local-zones-user-workload-ingress_{context}"]
+=== User-workload ingress traffic on AWS Local Zones
+
+Developers must use an external Load Balancer, such as AWS Application Load Balancers (ALB), to enable internet access to applications on AWS Local Zones. The ALB Operator is available through the Operator Lifecycle Manager (OLM).
```

Comment:
Better to use the full name `AWS Load Balancer Operator` so user can search it in OperatorHub easily.
```
The AWS Load Balancer Operator is available through the Operator Lifecycle Manager (OLM)
```